scripts/get-latest-rhocp-repo.sh

```diff
@@ -35,18 +31,9 @@ stop=$(( current_minor - 3 ))
 # and following code will try to access rhocp-4.15 (which is not released yet)
 # and then rhocp-4.14 (which will be returned from the script because it's usable).
 for ver in $(seq "${current_minor}" -1 "${stop}"); do
-    repository="https://cdn.redhat.com/content/dist/layered/rhel9/$(uname -m)/rhocp/4.${ver}"
-    exit_code=$(curl \
-        --silent \
-        --location \
-        --output /dev/null \
-        --write-out "%{http_code}" \
-        --cacert "${cacert}" \
-        --cert "${cert}" \
-        --key "${key}" \
-        "${repository}/os/repodata/repomd.xml")
-
-    if [[ "${exit_code}" == "200" ]]; then 
+    repository="rhocp-4.${ver}-for-rhel-9-$(uname -m)-rpms"
+    if sudo dnf -v repository-packages "${repository}" info cri-o 1>&2;
```

Comment:
@dhellmann, in the curl command, we wrote the output to /dev/null. I think this is necessary because we are using the output from the script to set versions. Otherwise, it's going to break the code that's using this script.